dali/pipeline/operators/displacement/displacement_filter_impl_gpu.cuh

```diff
@@ -370,18 +370,18 @@ class DisplacementFilter<GPUBackend, Displacement,
     const auto N = input.ntensor();
     const int pitch = nDims + 1;  // shape and offset
 
-    meta_cpu.Resize({N, pitch});
+    meta_cpu.Resize({static_cast<long>(N), pitch});
```

Comment:
Maybe Resize should be adjusted as well?

Reply:
Sure, I wanted to do a small change, but since you agree with my change, I'll make something bigger of it